pymeasure/instruments/keithley/keithley2700.py

```diff
@@ -359,3 +363,574 @@ def display_closed_channels(self):
+        validator=truncated_range,
+        values=[0, 999999.999]
+    )
+
+    ##########
+    # Buffer #
+    ##########
+
+    buffer_points = Instrument.control(
+        ":TRAC:POIN?", ":TRAC:POIN %d",
+        """ An integer property that controls the number of buffer points. This
+        does not represent actual points in the buffer, but the configuration
+        value instead. """,
+        validator=truncated_range,
+        values=[2, 55000],
+        cast=int
+    )
+    means = Instrument.measurement(
+        ":CALC2:FORM MEAN;:CALC2:IMM?;",
+        """ Reads the calculated means (averages) for voltage,
+        current, and resistance from the buffer data  as a list. """
+    )
+    maximums = Instrument.measurement(
+        ":CALC2:FORM MAX;:CALC2:IMM?;",
+        """ Returns the calculated maximums for voltage, current, and
+        resistance from the buffer data as a list. """
+    )
+    minimums = Instrument.measurement(
+        ":CALC2:FORM MIN;:CALC2:IMM?;",
+        """ Returns the calculated minimums for voltage, current, and
+        resistance from the buffer data as a list. """
+    )
+    standard_devs = Instrument.measurement(
+        ":CALC2:FORM SDEV;:CALC2:IMM?;",
+        """ Returns the calculated standard deviations for voltage,
+        current, and resistance from the buffer data as a list. """
+    )
+
+####
+
+    def measure_voltage(self, max_voltage=1, ac=False):
+        """ Configures the instrument to measure voltage,
+        based on a maximum voltage to set the range, and
+        a boolean flag to determine if DC or AC is required.
+
+        :param max_voltage: A voltage in Volts to set the voltage range
+        :param ac: False for DC voltage, and True for AC voltage
+        """
+        if ac:
+            self.mode = 'voltage ac'
+            self.voltage_ac_range = max_voltage
+        else:
+            self.mode = 'voltage'
+            self.voltage_range = max_voltage
+
+    def measure_current(self, max_current=10e-3, ac=False):
+        """ Configures the instrument to measure current,
+        based on a maximum current to set the range, and
+        a boolean flag to determine if DC or AC is required.
+
+        :param max_current: A current in Volts to set the current range
+        :param ac: False for DC current, and True for AC current
+        """
+        if ac:
+            self.mode = 'current ac'
+            self.current_ac_range = max_current
+        else:
+            self.mode = 'current'
+            self.current_range = max_current
+
+    def measure_resistance(self, max_resistance=10e6, wires=2):
+        """ Configures the instrument to measure voltage,
+        based on a maximum voltage to set the range, and
+        a boolean flag to determine if DC or AC is required.
+
+        :param max_voltage: A voltage in Volts to set the voltage range
+        :param ac: False for DC voltage, and True for AC voltage
```

Comment:
The docstring does not match to the content of the method. Wondering, what "wires" means, I stumbled upon the discrepancy.